tests/pr_test.go

```diff
@@ -180,3 +180,39 @@ func TestRunStandardUpgradeSolution(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestPlanValidation(t *testing.T) {
+	t.Parallel()
+
+	// Generate a 15 char long random string for the admin_pass
+	randomBytes := make([]byte, 13)
+	_, randErr := rand.Read(randomBytes)
+	require.Nil(t, randErr) // do not proceed if we can't gen a random password
```

Comment:
we are duplicating this code - suggest to add it as a global function and re-use it in different tests